tests/lib/ecma-features.js

```diff
@@ -93,7 +93,11 @@ describe("ecmaFeatures", function() {
 
                 result = ex;    // if an error is thrown, match the error
             }
-            assert.deepEqual(result, expected);
+            try {
+                assert.deepEqual(result, expected);
+            } catch (e) {
+                throw result;
```

Comment:
didn't mean to commit this, but it's really helpful for debugging. (will remove :))